lib/zendesk_apps_support/validations/marketplace.rb

```diff
@@ -14,6 +16,13 @@ def no_symlinks(path)
           end
           nil
         end
+
+        def no_experiments(manifest)
+          non_public_experiments = manifest.enabled_experiments - WHITELISTED_EXPERIMENTS
```

Comment:
that's user supplied input, not necessarily `non_public` but more like `invalid`?